src/Filter/Filter.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+namespace DocHeader\Filter;
+
+/**
+ * @author  Jefersson Nathan <malukenho@phpse.net>
+ * @license MIT
+ */
+final class Filter
+{
+    /**
+     * @var string
+     */
+    private $docheader;
+
+    private $defaultFilters = [
+        CurrentYear::class,
+    ];
+
+    /**
+     * @param string $docheader
+     */
+    public function __construct($docheader)
+    {
+        $this->docheader = $docheader;
+    }
+
+    /**
+     * @return string
+     */
+    public function applyFilters()
+    {
+        array_map(
+            function ($filterName) {
+                $this->docheader = (new $filterName)->__invoke($this->docheader);
```

Comment:
This assignment done in an `array_map()` call seems wrong. Also, `array_map()` is not returning anything?